[16.0][ADD] helpdesk_mgmt_template: add module (#614)
OCA/helpdesk: helpdesk_mgmt_template/readme/CONFIGURE.rst

> **To Configure Ticket's Description Template**
> - Go to Helpdesk-->Configuration-->Categories
> - Create a New Category or select an existing record
> - In the Template field pre-define the ticket's description that will be triggered based on the category selected
> 
> **To Configure Categories available for the Team**
> - Go to Helpdesk-->Configuration-->Categories
> - Create a New Team or select an existing record
> - Select categories available for the Team

Reviewer: ```suggestion
**To Configure Ticket's Description Template**
- Go to Helpdesk -> Configuration -> Categories.
- Create a new Category or select an existing Category.
- Set the *Template* field with pre-defined the ticket's description that will be triggered based on the category selected.

**To configure available Categories for Teams**
- Go to Helpdesk -> Configuration -> Categories.
- Create a new Team or select an existing Team.
- Select categories which are available for the Team.
```
